samples/javascript_typescript/60.multilingual-luis-bot/translator-middleware.ts

```diff
@@ -0,0 +1,51 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+"use strict";
+Object.defineProperty(exports, "__esModule", { value: true });
+
+import { Middleware, TurnContext, ConversationState, BotStatePropertyAccessor }  from 'botbuilder';
+import {MicrosoftTranslator, TranslateArrayOptions} from './translator';
+let _translator = null;
+
+const ENGLISH_LANGUAGE = 'en';
+const DEFAULT_LANGUAGE = ENGLISH_LANGUAGE;
+export class TranslatorMiddleware {
+    private translatorKey: string;
+    private to: string;
+    /**
+     * Creates a translation middleware.
+    *
+        @param {string} translatorKey Microsoft Text Translation API key.
+    *   @param {string} to 
+     */
+    constructor(to, translatorKey, noTranslatePatterns?: {[id: string] : string[]}, wordDictionary?: { [id: string]: string }) {
```

Comment:
```suggestion
    constructor(private to, private translatorKey, noTranslatePatterns?: {[id: string] : string[]}, wordDictionary?: { [id: string]: string }) {
```
You can do this and remove declaring the instance variables manually